projects/every-voice/src/lib/every-voice/every-voice.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, OnInit, Input } from "@angular/core";
+import { EveryVoiceService } from "../every-voice.service";
+
+@Component({
+  selector: "lib-every-voice",
+  templateUrl: "./every-voice.component.html",
+  styleUrls: ["./every-voice.component.css"],
+})
+export class EveryVoiceComponent implements OnInit {
+  @Input() textToGenerate: string;
+  @Input() icon = "volume_up";
+
```

Comment:
Should this component be visible if EveryVoiceService is disabled ?
```suggestion
@Input() stopIcon = "stop";
isPlaying = false;
```

Reply:
fair enough - I did this in the wordweaver code, but may as well actually just add `[hidden]` on the component itself